### PR TITLE
Add guest post on combining Flux and Pulumi

### PR DESCRIPTION
**Pull request by squaremo**
Drafted at https://hackmd.io/nBHp37rFQMaQU3qgOMj1eQ?view, now open for comments here :-)

### CODE REVIEW

**Comment by dholbach** [comment]
→ https://deploy-preview-1371--fluxcd.netlify.app/blog/

**Comment by dholbach** [comment]
Do we want to add a Pulumi logo or diagram or something else to the blog entry?

**Comment by squaremo** [comment]
> Do we want to add a Pulumi logo or diagram or something else to the blog entry?

Probably! I will find a good logo/diagram and a good spot.

**Comment by dholbach** [comment]
> > Do we want to add a Pulumi logo or diagram or something else to the blog entry?
> 
> Probably! I will find a good logo/diagram and a good spot.

https://github.com/fluxcd/website/tree/main/content/en/blog#adding-images

LMK if you need help.

File: content/en/blog/2023-02-08-flux-pulumi-superpowers.md [hunk 1]
```diff
@@ -0,0 +1,161 @@
...
+The support for Flux sources in the Pulumi operator gives you more ways to make your Pulumi programs
+available to the operator. Notably, you can now package your program as an OCI image and push it to
+an image registry, before using it with the operator. This might be appealing if you have deployment
+pipelines to generate Kubernetes YAML (e.g., from [Cue](https://cuelang.org/)) or other data, before
+using it in a program. It's also convenient when you're on a platform like AWS, GCP or Azure,
+because these have OCI registries with useful operational features like caching, security scanning,
+and so on.
+
+But I think an even better reason for using Flux is that it can verify your sources. If you use a
+Flux source in a Pulumi stack, you can take better secure your supply chain. When you're using an
```
Comment (aryan9600) [inline content/en/blog/2023-02-08-flux-pulumi-superpowers.md:36]:
```suggestion
Flux source in a Pulumi stack, you can take better care of your supply chain. When you're using an
```
or
```suggestion
Flux source in a Pulumi stack, you can secure your supply chain. When you're using an
```

**Review by aryan9600 — CHANGES_REQUESTED** [review]
its a really good blog post! found a minor typo :)

File: content/en/blog/2023-02-08-flux-pulumi-superpowers.md (hunk 1 above)
Comment (scottrigby) [reply content/en/blog/2023-02-08-flux-pulumi-superpowers.md:36]: I think we just want to remove the word "take" here. In context, Flux can better secure your supply chain by verifying cosign signatures.

```suggestion
Flux source in a Pulumi stack, you can better secure your supply chain. When you're using an
```

Comment (squaremo) [reply content/en/blog/2023-02-08-flux-pulumi-superpowers.md:36]: Agreed and agreed, thanks both.

**Comment by scottrigby** [comment]
@squaremo DCO bot is running into an issue [here](https://github.com/fluxcd/website/pull/1371/checks?check_run_id=11233178887):

> Commit sha: [448a521](https://github.com/fluxcd/website/pull/1371/commits/448a52177ded2941760a415643091e308bc4eb35), Author: Michael Bridgen, Committer: GitHub; Expected "Michael Bridgen [mbridgen@pulumi.com](mailto:mbridgen@pulumi.com)", but got "Michael Bridgen [mikeb@squaremobius.net](mailto:mikeb@squaremobius.net)".

I can set it to pass, because as far as DCO proper is concerned this is totally legit. Or would you rather rebase?

**Comment by squaremo** [comment]
> DCO bot is running into an issue

I reset the author and sign-off.

> 2. Adding an image

I would love to have a diagram in there, of Flux providing sources and Pulumi creating infrastructure, but my capacity for making one is pretty limited (in terms of skill and in terms of time, at present :-) As a fallback, I've added a Pulumi mascot with a cape, which seemed apposite.

**Comment by dholbach** [comment]
Thanks for doing that Michael!

Are you OK with me changing it so that the image is used as a social preview for the post? https://deploy-preview-1371--fluxcd.netlify.app/blog/

see https://github.com/fluxcd/website/compare/b0a6afc0ad1a5073ed70cd7c77156a7f36cd7c9b..dfb29082a74265241bb0d252532cd5add676d116

**Comment by dholbach** [comment]
if you want to do it, adding "feature" to the image name should be enough

**Comment by dholbach** [comment]
:rocket:

**Comment by dholbach** [comment]
Ok cool - let me apply some finishing touches and publish afterwards.

Thanks so much for your work on this @squaremo - this is great!

**Pull request merged**